utilities/pipelines/publish/Publish-ModuleFromPathToPBR.ps1

```diff
@@ -38,76 +38,95 @@ function Publish-ModuleFromPathToPBR {
     )
 
     # Load used functions
+    . (Join-Path $RepoRoot 'utilities' 'pipelines' 'publish' 'helper' 'Get-VersionedModuleList.ps1')
     . (Join-Path $RepoRoot 'utilities' 'pipelines' 'publish' 'helper' 'Get-ModulesToPublish.ps1')
     . (Join-Path $RepoRoot 'utilities' 'pipelines' 'publish' 'helper' 'Get-ModuleTargetVersion.ps1')
     . (Join-Path $RepoRoot 'utilities' 'pipelines' 'publish' 'helper' 'New-ModuleReleaseTag.ps1')
     . (Join-Path $RepoRoot 'utilities' 'pipelines' 'publish' 'helper' 'Get-ModuleReadmeLink.ps1')
     . (Join-Path $RepoRoot 'utilities' 'pipelines' 'sharedScripts' 'Get-BRMRepositoryName.ps1')
     . (Join-Path $RepoRoot 'utilities' 'pipelines' 'sharedScripts' 'tokenReplacement' 'Convert-TokensInFileList.ps1')
 
-    $moduleFolderPath = Split-Path $TemplateFilePath -Parent
-    $moduleBicepFilePath = Join-Path $moduleFolderPath 'main.bicep'
+    $topModuleFolderPath = Split-Path $TemplateFilePath -Parent
+    # $moduleFolderPath = Split-Path $TemplateFilePath -Parent
 
-    # 1. Test if module qualifies for publishing
-    if (-not (Get-ModulesToPublish -ModuleFolderPath $moduleFolderPath)) {
-        Write-Verbose 'No changes detected. Skipping publishing' -Verbose
-        return
-    }
```

Comment:
Would it not have been easier to keep this logic, but isntead of throwing the output of 'Get-ModulesToPublish' away, to instead fetch it, and for it's folder paths check if any contain a `version.json` file which would justify a publish? This filter could maybe even be implemented in the `Get-ModulesToPublish` function (using e.g., a switch parameter).

Reply:
Great point. Modified the logic to loop on the Get-ModulesToPublish output plus added minor adjustments. Please check